src/locales/pt/translation.json

```diff
@@ -0,0 +1,181 @@
+{
+  "1 Day": "1 Dia",
+  "1 Month": "1 Mês",
+  "1 Week": "1 Semana",
+  "2 Weeks": "2 Semanas",
+  "3 Days": "3 Dias",
+  "3 Months": "3 Meses",
+  "6 Months": "6 Meses",
+  "Accept": "Aceitar",
+  "Acknowledge": "Confirmar",
+  "acknowledged": "confirmado(s)",
+  "Acknowledged": "Confirmado(s)",
+  "Acknowledgments": "Confirmações",
+  "Actions": "Ações",
+  "Add Note": "Adicionar Nota",
+  "Add Note to incident(s) here": "Adicionar Nota a incidente(s) aqui",
+  "Add Responders": "Adicionar Respondentes",
+  "Alert Custom Detail Column Definitions": "Coluna de Definições dos Detalhes Personalizados do Alerta",
+  "Alert Custom Detail Column Definitions Placeholder": "Marcador de Coluna de Definições dos Detalhes Personalizados do Alerta",
+  "and their alerts have been merged onto incident": "e os seus alertas foram combinados no incidente",
+  "Assignees": "Detentores",
```

Comment:
Please change to: Responsáveis (I had originally reserved "responsáveis" for the "responders" translation but, in the end, went ahead with the same as brazilian: "detentores" however, if anyone has a better suggestion for reponders in pt, please add it to this file.)